File: Cpp/Program/Produk.cpp
#include <iostream>
#include <string>
#include <iomanip>
#include <vector>
#include <limits>
using namespace std;

class Produk {
private:
    string namaProduk;
    string merkProduk;
    string noSeriProduk;
    long hargaProduk;
    string deskripsi;

public:
    Produk() {
        namaProduk = "";
        merkProduk = "";
        noSeriProduk = "";
        hargaProduk = 0;
        deskripsi = "";
    }

    void setNamaProduk(string n) { namaProduk = n; }
    void setMerkProduk(string m) { merkProduk = m; }
    void setNoSeriProduk(string no) { noSeriProduk = no; }
    void setHargaProduk(long h) { hargaProduk = h; }
    void setDeskripsi(string d) { deskripsi = d; }

    string getNamaProduk() { return namaProduk; }
    string getMerkProduk() { return merkProduk; }
    string getNoSeriProduk() { return noSeriProduk; }
    long getHargaProduk() { return hargaProduk; }
    string getDeskripsi() { return deskripsi; }
};

File: Java/Program/Produk.java
public class Produk {
    private String namaProduk, merkProduk, noSeriProduk, deskripsi;
    private double hargaProduk;

    // Tambahkan konstruktor kosong
    public Produk() {}

    // Getter dan Setter
    public String getNamaProduk() { return namaProduk; }
    public void setNamaProduk(String v) { namaProduk = v; }

    public String getMerkProduk() { return merkProduk; }
    public void setMerkProduk(String v) { merkProduk = v; }

    public String getNoSeriProduk() { return noSeriProduk; }
    public void setNoSeriProduk(String v) { noSeriProduk = v; }

    public double getHargaProduk() { return hargaProduk; }
    public void setHargaProduk(double v) { hargaProduk = v; }

    public String getDeskripsi() { return deskripsi; }
    public void setDeskripsi(String v) { deskripsi = v; }
}